libs/wallet/src/web3modal/web3ModalConfig.ts

```diff
@@ -0,0 +1,49 @@
+import { CHAIN_INFO, RPC_URLS } from '@cowprotocol/common-const'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+import { createWeb3Modal, defaultConfig } from '@web3modal/ethers5/react'
+import { addSafeAppConnector } from './connectors/safeAppConnector'
+import { addTrezorConnector } from './connectors/trezor/trezorConnector'
+
+const projectId = '7bd490257c3457cc6a5ebcc02066c21a'
+
+const metadata = {
+  name: 'CoW Swap',
+  description: 'CoW Swap | The smartest way to trade cryptocurrencies',
+  url: 'https://swap.cow.fi',
+  icons: ['https://swap.cow.fi/favicon.png'],
+}
+
+const ethersConfig = defaultConfig({
+  metadata,
+
+  enableEIP6963: true,
+  enableInjected: true,
+  enableCoinbase: true,
+  rpcUrl: '...', // used for the Coinbase SDK
```

Comment:
I think I mentioned this somewhere else.
If we are using it, shouldn't it be set?